modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt

```diff
@@ -33,25 +34,26 @@ import jp.co.soramitsu.iroha2.generated.datamodel.permissions.PermissionToken
 import java.math.BigDecimal
 import jp.co.soramitsu.iroha2.generated.datamodel.account.Id as AccountId
 import jp.co.soramitsu.iroha2.generated.datamodel.asset.Id as AssetId
+import jp.co.soramitsu.iroha2.generated.datamodel.domain.Id as DomainId
 import jp.co.soramitsu.iroha2.generated.datamodel.peer.Id as PeerId
 
-const val CAN_SET_KEY_VALUE_USER_ASSETS_TOKEN = "can_set_key_value_in_user_assets"
-const val CAN_REMOVE_KEY_VALUE_IN_USER_ASSETS = "can_remove_key_value_in_user_assets"
-const val CAN_SET_KEY_VALUE_IN_USER_METADATA = "can_set_key_value_in_user_metadata"
-const val CAN_REMOVE_KEY_VALUE_IN_USER_METADATA = "can_remove_key_value_in_user_metadata"
-const val CAN_SET_KEY_VALUE_IN_ASSET_DEFINITION = "can_set_key_value_in_asset_definition"
-const val CAN_REMOVE_KEY_VALUE_IN_ASSET_DEFINITION = "can_remove_key_value_in_asset_definition"
-const val CAN_MINT_USER_ASSET_DEFINITIONS_TOKEN = "can_mint_user_asset_definitions"
-const val CAN_MINT_USER_ASSETS_DEFINITION = "can_mint_user_asset_definitions"
-const val CAN_BURN_ASSET_WITH_DEFINITION = "can_burn_asset_with_definition"
-const val CAN_BURN_USER_ASSETS_TOKEN = "can_burn_user_assets"
-const val CAN_REGISTER_DOMAINS_TOKEN = "can_register_domains"
-const val CAN_TRANSFER_USER_ASSETS_TOKEN = "can_transfer_user_assets"
-const val CAN_UNREGISTER_ASSET_WITH_DEFINITION = "can_unregister_asset_with_definition"
-
-const val ACCOUNT_ID_TOKEN_PARAM_NAME = "account_id"
-const val ASSET_ID_TOKEN_PARAM_NAME = "asset_id"
-const val ASSET_DEFINITION_PARAM_NAME = "asset_definition_id"
+val CAN_SET_KEY_VALUE_USER_ASSETS_TOKEN = "can_set_key_value_in_user_assets".asName()
```

Comment:
Let's make all them lazy to not load all them in memory
```suggestion
val CAN_SET_KEY_VALUE_USER_ASSETS_TOKEN = by lazy { "can_set_key_value_in_user_assets".asName() }
```